test/server/lib/model.js

```diff
@@ -66,5 +67,93 @@ describe('lib/model', () => {
       });
     });
 
+    describe('dayCounts', () => {
+      const dayCounts = model.__get__('dayCounts');
+
+      it('should take start date "2020-03-11", end date "2020-03-11" and return "{week:1,end:0}"', () =>
+        expect(dayCounts('2020-03-11', '2020-03-11')).to.deep.equal([1, 0]));
+      it('should take start date "2020-03-14", end date "2020-03-14" and return "{week:0,end:1}"', () =>
+        expect(dayCounts('2020-03-14', '2020-03-14')).to.deep.equal([0, 1]));
+      it('should take start date "2020-03-11", end date "2020-03-12" and return "{week:2,end:0}"', () =>
+        expect(dayCounts('2020-03-11', '2020-03-12')).to.deep.equal([2, 0]));
+      it('should take start date "2020-03-14", end date "2020-03-15" and return "{week:0,end:2}"', () =>
+        expect(dayCounts('2020-03-14', '2020-03-15')).to.deep.equal([0, 2]));
+      it('should take start date "2020-03-09", end date "2020-03-15" and return "{week:0,end:2}"', () =>
+        expect(dayCounts('2020-03-09', '2020-03-15')).to.deep.equal([5, 2]));
+    });
+
+    describe('hours', () => {
+      const hours = model.__get__('hours');
+
+      it('should return an array of 24 items', () =>
+        expect(hours('title'))
+          .to.be.an('object').that.has.keys(['name', 'data'])
+          .and.property('data').to.be.an('array').that.has.lengthOf(24)
+      );
+
+      it('should have an object for each hour of the day, with a count of 0', () =>
+        hours().data.forEach((h, n) => expect(h)
+          .to.deep.equal({ count: 0, hour: n}))
+      );
+    });
+  });
+
+  describe('hourlyUsage', () => {
+    const stub = sinon.stub();
+    const { hourlyUsage } = proxyquire('../../../src/lib/model', {
+      './db/query': { hourlyUsage: stub }
+    });
```

Comment:
Could this be moved to the top?

Reply:
I mean it _could_ but it's only used in this scope, so why bother polluting the parent scopes with it?

Reply:
Moved to top. Fixed

Reply:
Sorry I missed this and should have replied to your comment. The reason I asked is that I think it's nice to declare dependencies in one place. I think dynamic imports should be reserved for times when you really need them.

Reply:
Just noticed that this code seems not to have changed.

Reply:
Yup, works for me. Fixed